controllers/usersignupcleanup/usersignup_cleanup_controller_test.go

```diff
@@ -251,7 +263,22 @@ func TestUserCleanup(t *testing.T) {
 		require.NoError(t, err)
 		assert.True(t, deleted)
 	})
+}
 
+func alreadyDeletedSignupIgnored(t *testing.T, userSignup *toolchainv1alpha1.UserSignup) {
+	// Now let's simulate the situation when the signup is already being deleted and reconcile it again.
```

Comment:
could we also check that the Prom metric is not incremented again?

Reply:
Yep. Missed that. Fixed here: https://github.com/codeready-toolchain/host-operator/pull/881/commits/672d77306b84faec9201d088c45881b59410bd9b
Thanks.

Reply:
thanks!